Project/src/test/java/com/github/lgooddatepicker/TestGithubIssues.java

```diff
@@ -65,12 +65,7 @@ public void TestIssue82() throws InterruptedException {
     final ExceptionInfo exInfo = new ExceptionInfo();
     try {
       TestHelpers.registerUncaughtExceptionHandlerToAllThreads(
-          new Thread.UncaughtExceptionHandler() {
-            @Override
-            public void uncaughtException(Thread t, Throwable e) {
-              exInfo.set(t.getName(), e);
-            }
-          });
+          (t, e) -> exInfo.set(t.getName(), e));
```

Comment:
I'm guessing you are using an automatic refactoring here. That tool seems to simply use the parameter names from the anonymous class. 
When using lambda functions the parameter types are not visible anymore and the names have to be more concise to still have meaning. Sadly this cannot be done by any tools.

I would like it much better if the variable names would be `thread` and `throwable` here.

Reply:
done